SU2_CFD/include/numerics/CNumerics.hpp

```diff
@@ -1509,16 +1509,48 @@ class CNumerics {
    */
   static void tql2(su2double **V, su2double *d, su2double *e, unsigned short n);
 
+  /*!
+   * \brief Set the pressure derivatives.
+   * \param[in] val_dPdU_i - pressure derivatives at i. 
+   * \param[in] val_dPdU_j - pressure derivatives at j.
+   */
   virtual inline void SetdPdU(su2double *val_dPdU_i, su2double *val_dPdU_j)       { }
 
+  /*!
+   * \brief Set the temperature derivatives.
+   * \param[in] val_dTdU_i - temperature derivatives at i. 
+   * \param[in] val_dTdU_j - temperature derivatives at j.
+   */
   virtual inline void SetdTdU(su2double *val_dTdU_i, su2double *val_dTdU_j)       { }
 
+  /*!
+   * \brief Set the vib-el temperture derivatives.
+   * \param[in] val_dTvedU_i - t_ve derivatives at i. 
+   * \param[in] val_dTvedU_j - t_ve derivatives at j.
+   */
   virtual inline void SetdTvedU(su2double *val_dTvedU_i, su2double *val_dTvedU_j) { }
 
+  /*!
+   * \brief Set the vib-elec energy.
+   * \param[in] val_Eve_i - vib-el energy at i. 
+   * \param[in] val_Eve_j - vib-el energy at j.
+   */
   virtual inline void SetEve(su2double *val_Eve_i, su2double *val_Eve_j)          { }
 
+  /*!
+   * \brief Set the vib-elec specific heat.
+   * \param[in] val_Cvve_i - Cvve at i. 
+   * \param[in] val_Cvve_j - Cvve at j.
+   */
   virtual inline void SetCvve(su2double *val_Cvve_i, su2double *val_Cvve_j)       { }
 
+  /*!
+   * \brief Set the ratio of specific heats.
+   * \param[in] val_Gamma_i - Gamma at i.
+   * \param[in] val_Gamma_j - Gamma at j.
+   */
+  virtual inline void SetGamma(su2double val_Gamma_i, su2double val_Gamma_j)       { }
```

Comment:
Are there tabs in these files? Maybe you can run clang-format at some point (the idea is to eventually run in on the entire code base).

Reply:
I only use spaces.   But there may be tabs in the file.....I can run Clang-format to find them.